source/administration/statistics.rst

```diff
@@ -19,7 +19,7 @@ Total Teams
     The total number of teams created on your system.
 
 Total Channels
-    The total number of public channels and private groups created in all the teams on your system, including deleted channels. Doesn't include direct message channels.
+    The total number of public channels and private channels created in all the teams on your system, including deleted channels. Doesn't include direct message channels.
```

Comment:
"The total number of public channels and private channels created ..." > "The total number of channels created..."?